src/components/radio-button/radio-button.styles.ts

```diff
@@ -0,0 +1,55 @@
+import { useTheme } from 'native-base';
+import { StyleSheet } from 'react-native';
+
+import { RadioButtonKind } from './radio-button.types';
+
+type RadioKind = 'primary' | 'success' | 'error';
+
+interface RadioKindStyle {
+  borderColor: string;
+  innerColor: string;
+}
+
+export const useRadioKindStyles = (): Record<RadioKind, RadioKindStyle> => {
+  const theme = useTheme();
+
+  return {
+    [RadioButtonKind.PRIMARY]: {
+      borderColor: theme.colors.primary['500'],
+      innerColor: theme.colors.primary['600'],
+    },
+    [RadioButtonKind.SUCCESS]: {
+      borderColor: theme.colors.success['500'],
+      innerColor: theme.colors.success['600'],
+    },
+    [RadioButtonKind.ERROR]: {
+      borderColor: theme.colors.danger['600'],
+      innerColor: theme.colors.danger['500'],
+    },
+  };
+};
+
+export const styles = StyleSheet.create({
```

Comment:
There should almost be no value that should be hard coded and come from theme. For ex: paddingVertical.

This way when we make a change at theme level, all components changes accordingly.